src/components/index.js

```diff
@@ -22,5 +22,6 @@ import { FirstVedioCarousel } from "./Carousel/FirstVedioCarousel" ;
 import { MainVedio } from "./Carousel/MainVedio";
 import { Carousel } from "./Carousel/Carousel";
 import { GridTemplate } from "./Grid-template/GridTemplate";
+import { SingleProductDetails } from "./Single-product-detail/SingleProductDetails";
 
-export { Header, RoutesComponent, CategoryType, ShopNowImgSection, NewArrivalsSection, DiscountedProducts, ProductsOnSale, Footer, SideBar, ProductsSection, ProductCard, PopUpFiltersContainer, FilterableForm, WishlistProduct, ProductsInCart, LogInCard, LogOutCard, SignUpCard, AprizeLogo, CombinedProvider, FirstVedioCarousel, MainVedio, Carousel, GridTemplate };
\ No newline at end of file
+export { Header, RoutesComponent, CategoryType, ShopNowImgSection, NewArrivalsSection, DiscountedProducts, ProductsOnSale, Footer, SideBar, ProductsSection, ProductCardListing, PopUpFiltersContainer, FilterableForm, WishlistProduct, ProductsInCart, LogInCard, LogOutCard, SignUpCard, AprizeLogo, CombinedProvider, FirstVedioCarousel, MainVedio, Carousel, GridTemplate, SingleProductDetails };
```

Comment:
You can directly export the components without importing them first, as follows.
```js
export { Footer } from "./Footer/Footer";
```